test/extended/util/annotate/rules.go

```diff
@@ -25,6 +25,7 @@ var (
 			`\[Feature:IPv6DualStack.*\]`,
 			`version v1 should create Endpoints and EndpointSlices for Pods matching a Service`, // off by default.
 			`\[Feature:ImmutableEphemeralVolume\]`,                                              // flag gate is off
+			`\[Feature:ServiceAccountIssuerDiscovery\]`,                                         // flag gate is off
```

Comment:
the PR in o/api plus a follow-up in ckoa will fix this?

Reply:
It's not broken, it's disabled by default as per the comment. The failure of the test was due to the feature not being enabled, not due to incompatiblity. @stlaz looked into the failure and realized the default issuer we were setting wouldn't be compatible with the feature should it ever be enabled.

Reply:
I understand.

Reply:
Added bz as per comment about other disabled test: https://bugzilla.redhat.com/show_bug.cgi?id=1820423